tests/try_scraper/test_try_scraper.py

```diff
@@ -6,73 +6,55 @@
 
 import os
 import urllib
-
+import pytest
 from mock import patch
-
 from mozdownload import TryScraper
 from mozdownload.utils import urljoin
-import mozhttpd_base_test as mhttpd
 
-firefox_tests = [
+@pytest.mark.parametrize("args,filename,url", [
     # -p mac64 --revision=8fcac92cfcad
-    {'args': {'platform': 'mac64',
-              'revision': '8fcac92cfcad'},
-     'filename': '8fcac92cfcad-firefox-38.0a1.en-US.mac.dmg',
-     'url': 'firefox/try-builds/test-user@mozilla.com-8fcac92cfcad/'
-            'try-macosx64/firefox-38.0a1.en-US.mac.dmg'},
+    ({'platform': 'mac64', 'revision': '8fcac92cfcad'},
+     '8fcac92cfcad-firefox-38.0a1.en-US.mac.dmg',
+     'firefox/try-builds/test-user@mozilla.com-8fcac92cfcad/'
+     'try-macosx64/firefox-38.0a1.en-US.mac.dmg'),
```

Comment:
Would the `url` fit into a single line now by not reaching the 100 char limit? We don't have that much indentation anymore, and would be helpful for readability.